src/diff.ts

```diff
@@ -160,13 +160,12 @@ export function summary(
   role: string,
   projectId: string
 ): string {
-  let diffContent
-  if (sql === '') {
-    diffContent = 'No schema changes detected'
-  } else {
-    diffContent = `\`\`\`diff\n${sql}\n\`\`\``
+  if (sql.trim() === '') {
```

Comment:
Have you checked if the `createPatch` returns an empty string? I thought it used to return something like

```diff
--- Database: neondb	(Branch: br-restless-cherry-a5wflpk7)
+++ Database: neondb	(Branch: br-lingering-surf-a520rn8j)
```

Reply:
If there are no changes we do an early return

```
  if (compareSchema.data?.sql === baseSchema.data?.sql) {
    return {
      sql: '',
      hash: '',
      compareBranch,
      baseBranch,
      role: username,
      database: database
    }
  }
```

Reply:
Right this makes sense.